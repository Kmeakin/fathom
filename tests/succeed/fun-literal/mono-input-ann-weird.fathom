fun (a : U32 : Type : Type) => a
