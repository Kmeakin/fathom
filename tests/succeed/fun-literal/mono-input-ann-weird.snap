stdout = '''
fun a => a : fun (a : U32) -> U32
'''
stderr = ''
